invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/document.js

```diff
@@ -108,6 +109,16 @@ class QueryBuilder {
     return this;
   }
 
+  pendingOverdue() {
+    const query = [
+      'circulation.has_items_for_loan:0',
```

Comment:
I would add `had_items:>0`, because otherwise you might show also Documents that have 0 items

Reply:
same for overbooked if applies